general/mem_manager.hpp

```diff
@@ -114,7 +114,8 @@ class Memory
       VALID_HOST    = 16,  ///< Host pointer is valid
       VALID_DEVICE  = 32,  ///< Device pointer is valid
       ALIAS         = 64,
-      EXEC_FLAG     = 128  ///< Optional execution flag used by MFEM
+      /// Execution flag reserved for use by MFEM, e.g. see Vector::UseDevice()
```

Comment:
If I'm correct this flag is only used by the macro `MFEM_FORALLIF` in the `Vector` class. And this flag is irrelevant for standard `MFEM_FORALL` which can be very confusing.